Cargo.lock

```diff
@@ -1222,7 +1238,7 @@ dependencies = [
  "base64 0.13.1",
  "bitcoin",
  "borsh",
- "brotli",
+ "brotli 3.5.0",
```

Comment:
Maybe it's time for us to update brotli to 6?

Reply:
Not in this PR probably